python.d/traefik.chart.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+# Description: traefik netdata python.d module
+# Author: Alexandre Menezes (@ale_menezes)
+
+from json import loads
+from collections import defaultdict
+from bases.FrameworkServices.UrlService import UrlService
+
+# default module values (can be overridden per job in `config`)
+update_every = 1
+priority = 60000
+retries = 10
+
+# charts order (can be overridden if you want less charts, or different order)
+ORDER = [
+    'response_statuses',
+    'response_codes',
+    'detailed_response_codes',
+    'requests',
+    'total_response_time',
+    'average_response_time',
+    'average_response_time_per_iteration',
+    'uptime'
+]
+
+CHARTS = {
+    'response_statuses': {
+        'options': [None, 'Response statuses', 'requests/s', 'responses', 'traefik.response_statuses', 'stacked'],
+        'lines': [
+            ['successful_requests', 'success', 'incremental'],
+            ['server_errors', 'error', 'incremental'],
+            ['redirects', 'redirect', 'incremental'],
+            ['bad_requests', 'bad', 'incremental'],
+            ['other_requests', 'other', 'incremental']
+        ]},
+    'response_codes': {
+        'options': [None, 'Responses by codes', 'requests/s', 'responses', 'traefik.response_codes', 'stacked'],
+        'lines': [
+            ['2xx', None, 'incremental'],
+            ['5xx', None, 'incremental'],
+            ['3xx', None, 'incremental'],
+            ['4xx', None, 'incremental'],
+            ['1xx', None, 'incremental'],
+            ['other', None, 'incremental']
+        ]},
+    'detailed_response_codes': {
+        'options': [None, 'Detailed response codes', 'requests/s', 'responses', 'traefik.detailed_response_codes', 'stacked'],
+        'lines': [
+        ]},
+    'requests': {
+        'options': [None, 'Requests', 'requests/s', 'requests', 'traefik.requests', 'line'],
+        'lines': [
+            ['total_count', 'requests', 'incremental']
+        ]},
+    'total_response_time': {
+        'options': [None, 'Total response time', 'seconds', 'timings', 'traefik.total_response_time', 'line'],
+        'lines': [
+            ['total_response_time_sec', None, 'absolute', 1, 1]
```

Comment:
`None` => `response`
Are you sure we need `total_response_time_sec` as absolute value. Does it makes any sense?

I think we can change `absolute` to `incremental`. And convert seconds to ms.
btw you have to multiply it in the code and set a divisor in chart.

In that case this will be the same chart as `average_response_time`.

Reply:
@l2isbad `total_response_time_sec` it's a sum of all response time. I believe that time correspond in all time that traefik works.

See the images below with traefik health dashboard and netdata corresponding chart.

traefik dashboard
![traefik health dashboard](https://www.dropbox.com/s/sfajh3wf6rg7iq1/traefik_dashboard.png?raw=1)

netdata sample
![netdata sample](https://www.dropbox.com/s/7ccthf8brz57n9c/netdata_ex1.png?raw=1)